UnicodeDecodeError on Windows for schema v140

```
E       UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 31972: character maps to <undefined>

C:\[...]\AppData\Local\Programs\Python\Python38\lib\encodings\cp1252.py:23: UnicodeDecodeError
```
I work at home on a Windows machine (Linux at RLI). I get above error in my tests for every test that tries to import schema v140 in branch . On Linux its running perfect on branch "feature/add-required-properties", succeeded travis tests in #11 proof that. I will look into it!

--- a/metadata/v130/example.py
+++ b/metadata/v130/example.py
@@ -1,5 +1,5 @@
 import json
 import os
 
-with open(os.path.join(os.path.dirname(__file__), "example.json"), "r") as f:
+with open(os.path.join(os.path.dirname(__file__), "example.json"), "rb") as f:
     METADATA_V130_EXAMPLE = json.loads(f.read())

--- a/metadata/v130/schema.py
+++ b/metadata/v130/schema.py
@@ -1,5 +1,5 @@
 import json
 import os
 
-with open(os.path.join(os.path.dirname(__file__), "schema.json"), "r") as f:
+with open(os.path.join(os.path.dirname(__file__), "schema.json"), "rb") as f:
     METADATA_V130_SCHEMA = json.loads(f.read())

--- a/metadata/v130/template.py
+++ b/metadata/v130/template.py
@@ -1,5 +1,5 @@
 import json
 import os
 
-with open(os.path.join(os.path.dirname(__file__), "template.json"), "r") as f:
+with open(os.path.join(os.path.dirname(__file__), "template.json"), "rb") as f:
     METADATA_V130_TEMPLATE = json.loads(f.read())

--- a/metadata/v140/example.py
+++ b/metadata/v140/example.py
@@ -1,5 +1,5 @@
 import json
 import os
 
-with open(os.path.join(os.path.dirname(__file__), "example.json"), "r") as f:
+with open(os.path.join(os.path.dirname(__file__), "example.json"), "rb") as f:
     METADATA_V140_EXAMPLE = json.loads(f.read())

--- a/metadata/v140/schema.py
+++ b/metadata/v140/schema.py
@@ -1,5 +1,5 @@
 import json
 import os
 
-with open(os.path.join(os.path.dirname(__file__), "schema.json"), "r") as f:
+with open(os.path.join(os.path.dirname(__file__), "schema.json"), "rb") as f:
     METADATA_V140_SCHEMA = json.loads(f.read())

--- a/metadata/v140/template.py
+++ b/metadata/v140/template.py
@@ -1,5 +1,5 @@
 import json
 import os
 
-with open(os.path.join(os.path.dirname(__file__), "template.json"), "r") as f:
+with open(os.path.join(os.path.dirname(__file__), "template.json"), "rb") as f:
     METADATA_V140_TEMPLATE = json.loads(f.read())

